src/web/waiters/OperationsWaiter.mjs

```diff
@@ -265,11 +313,43 @@ class OperationsWaiter {
             this.removeIntent = false;
         }.bind(this));
 
-        $("#edit-favourites-list [data-toggle=popover]").popover();
+        if (window.innerWidth >= this.app.breakpoint) {
+            $("#edit-favourites-list [data-toggle=popover]").popover();
+        }
         $("#favourites-modal").modal();
     }
 
 
+    /**
+     * Open operations dropdown
+     */
+    openOperationsDropdown() {
+        // the 'close' ( dropdown ) icon in Operations component mobile UI
+        const closeOperationsDropdown = document.getElementById("close-operations-dropdown");
+        const categories = document.getElementById("categories");
+
+        this.app.updateVisibility(categories, true);
+        this.app.updateVisibility(closeOperationsDropdown, true);
+    }
+
+
+    /**
+     * Hide any operation lists ( #categories or #search-results ) and the close-operations-dropdown
+     * icon itself, clear any search input
+     */
+    closeOperationsDropdown() {
+        const search = document.getElementById("search");
+
+        // if any input remains in #search, clear it
+        if (search.value.length) {
+            search.value = "";
+        }
+
+        this.app.updateVisibility(document.getElementById("categories"), false);
```

Comment:
Might be nice to use the same pattern as in `openOperationsDropdown()` where you first collect the element and store it in a const before passing it to `updateVisibility()`. Could also update `openOperationsDropdown()` to reflect the pattern in `closeOperationsDropdown()`. I have no opinion on which I like best. One is more verbose and readable, the other more concise.

Reply:
I see what you're saying but since I'm only grabbing each element once, I prefer not to store it in a const first